src/idl_gen_lobster.cpp

```diff
@@ -78,6 +78,16 @@ class LobsterGenerator : public BaseGenerator {
     return "int";
   }
 
+  std::string LobsterReturnType(const Type &type) {
```

Comment:
This function is identical to the one above, except for handling bool/string. Either figure out if the uses of `LobsterType` can also benefit from these types (best), or add an `is_return = false` argument.